doc/Learn/Markup/SourceUsage.md

```diff
@@ -0,0 +1,177 @@
+---
+uid: Uno.Extensions.Markup.SourceUsage
+---
+# Source and Relative Source
+
+Sometimes, when working with binding expressions, specifying the source or a relative source becomes necessary. In this section, we'll explore how to do that using strongly typed sources.
+
+## Source Binding
+
+Binding a property directly from one UI element to another is a common scenario. Here's how you can bind the Text property of a TextBox to the Content property of a button, and vice versa:
+
+```cs
+public partial class MainPage : Page
+{
+    public MainPage()
+    {
+        new StackPanel()
+            .Children(
+                new Button()
+                    .Assign(out var button)
+                    .Content("I am a button"),
+
+                // Creating a TextBox and binding its Text property to a Button's Content
+
+                new TextBox()
+                    .Text(x => x.Source(button)
+                                .Binding(() => button.Content));
+                    )
+    }
+}
+
+```
+
+```cs
+public partial class MainPage : Page
+{
+    public MainPage()
+    {
+        new StackPanel()
+            .Children(
+                new TextBox()
+                    .Text("I am a TextBox"),
+
+                // Creating a Button and binding its Content property to the TextBox's Text property with TwoWay binding
+
+                new Button()
+                    .Content(x => x.Source(textbox)
+                                .Binding(() => textbox.Text)
+                                .TwoWay());
+                    )
+    }
+}
+
+```
```

Comment:
```suggestion
```csharp
new StackPanel()
    .Children(
        new TextBox()
            .Assign(out var textBox)
            .Text("I am a TextBox"),

        // Creating a Button and binding its Content property to the TextBox's Text property with TwoWay binding
        new Button()
            .Content(x => x.Source(textBox)
                           .Binding(() => textBox.Text)
                           .TwoWay())
    )
```
```